backend/typescript/services/interfaces/taskService.ts

```diff
@@ -68,6 +68,30 @@ export interface ITaskService {
    */
   getTasksByAssignerId(assignerId: number): Promise<TaskDTO[]>;
 
+  /**
+   * Get all tasks by a start date
+   * @param startDate start date
+   * @returns a TaskDTO with task's information
+   * @throws Error if task retrieval fails
+   */
+  getTasksByStartDate(startDate: Date): Promise<TaskDTO[]>;
+
+  /**
+   * Get all tasks by an end date
+   * @param endDate end date
+   * @returns a TaskDTO with task's information
+   * @throws Error if task retrieval fails
+   */
+  getTasksByEndDate(endDate: Date): Promise<TaskDTO[]>;
+
+  /**
+   * Get all tasks by a status
+   * @param status status
+   * @returns a TaskDTO with task's information
```

Comment:
Shouldn't this be a list of TaskDTOs and maybe include details about what type of TaskDTOs are in the list? Same with line 74 and 82.

Reply:
Thanks! Added those changes